README.md

```diff
@@ -0,0 +1,153 @@
+pod 'Relativity', '~> 0.8.0'
+```
+
+### Carthage
+
+To install Relativity in your iOS project with [Carthage](https://github.com/Carthage/Carthage), add the following to your `Cartfile`:
+
+```ogdl
+github "dfed/Relativity"
+```
+
+Run `carthage` to build the framework and drag the built `Relativity.framework` into your Xcode project.
+
+### Swift Package Manager
+
+To install Relativity in your iOS project with [Swift Package Manager](https://github.com/apple/swift-package-manager), the following definition can be added to the dependencies of your `Project`:
+
+```swift
+  .Package(url: "https://github.com/dfed/Relativity.git", majorVersion: 0, minor: 8),
+```
+
+### Submodules
+
+To use git submodules, checkout the submodule with `git submodule add git@github.com:dfed/Relativity.git`, drag Relativity.xcodeproj to your project, and add Relativity as a build dependency.
+
+## Usage
+
+### Programmatic Layout
+
+There are four basic steps to laying out views programmatically. In `init` or `viewDidLoad`:
+1. Create your views.
+2. Create your view hierarchy with `addSubview()`.
+Then in `layoutSubviews` for a `UIView` or `viewDidLayoutSubviews` for a `UIViewController`:
+3. Size your views.
+4. Position your views.
+
+Note that with the above steps your UI will animate nicely during screen rotations and status bar size changes.
+
+### Anchoring UIViews
+
+Positioning views are done by anchoring a corner or edge of a view to another view’s corner or edge, plus an offset. A `-->` moves the view on the left side to the position described on the right side. A `<--` moves the view on the right side to the position described on the left side. You can also explicitly align views using the `align(to:xOffset:yOffset)` method in [ViewPosition.swift](Sources/ViewPosition.swift#L105).
+
+#### Examples
+
+To align view `a` to be 10 points to the left of view `b`:
+
+```swift
+  a.rightCenter --> 10.horizontalOffset + b.leftCenter
+```
+
+To align view `a` to be 10 points to the right of view `b`:
+
+```swift
+  b.rightCenter + 10.horizontalOffset <-- a.leftCenter
+```
+
+To align view `a` to be 10 points below its superview’s top center:
+
+```swift
+  a.topCenter --> 10.verticalOffset + .topCenter
+```
+
+For more examples, check out the [ViewPositionVisualization](RelativityVisualization.playground/Pages/ViewPositionVisualization.xcplaygroundpage/Contents.swift) playground page.
+
+#### Anchoring UILabels
+
+Relativity makes it easy to position your `UILabel`s to your designer’s spec. Design teams (and design products like [Sketch](https://www.sketchapp.com) and [Zeplin](https://zeplin.io)) measure the vertical distance to a label using the font’s [cap height](https://en.wikipedia.org/wiki/Cap_height) and [baseline](https://en.wikipedia.org/wiki/Baseline_(typography)). Relativity’s `align` methods measure `UILabel` distances the same way. So if your spec says that label `b` should be eight vertical points below label `a`, all you need is:
```

Comment:
Thought: Using the cap height and baseline is probably the correct way to have done this originally, but there's a lot of existing code using the metrics, and design teams that have grown accustomed to those metrics. It might be worth having either a per-align, or a global option (or both) that specifies which metrics to use so that migrations are easier.